agents-api/config/settings.py

```
@@ -4,50 +4,63 @@
-    "http://127.0.0.1:3000",
-    "https://localhost:3000",
-    "https://cur8t.com",
-    "https://www.cur8t.com",
-    "https://agents.cur8t.com"
+        "http://localhost:3000",
+        "http://127.0.0.1:3000",
+        "https://localhost:3000",
+        "https://cur8t.com",
+        "https://www.cur8t.com",
+        "https://agents.cur8t.com",
     ]
```

Ruff (0.13.3)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)